registration-processor-default.properties

```diff
@@ -984,3 +984,5 @@ mosip.registration.processor.lostrid.max-registration-date-filter-interval=30
 
 ##Ability to update contact information as part of LOST UIN
 uingenerator.lost.packet.allowed.update.fields=phone,email,permanentAddress
+logging.level.io.mosip.registration.processor.core.eventbus.KafkaMosipEventBus=DEBUG
+registration.processor.healthcheck.timeout=5000
```

Comment:
Lets rename this property to mosip.regproc.health-check.handler-timeout

Reply:
fixed